storybook/src/stories/UseDeepCompareEffect/API/countryCodes.json

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
I think it'd be best to use the country codes as the select input values, and the names as their labels. That way, the story component would no longer have to refer to this object to get the code from the name since it'll be passed in as a prop.

[Here](https://storybook.js.org/docs/react/essentials/controls#dealing-with-complex-values) are the storybook control docs on that. You'd have to swap the entries and make the codes the keys and the names the values, but I think that's a bit more conventional anyway.

Reply:
Also don't mind naming the file just `countries.json` and importing it as such